internal/pkg/cli/pipeline_delete.go

```diff
@@ -144,6 +160,22 @@ func (o *deletePipelineOpts) Execute() error {
 	return nil
 }
 
+type askDeployedPipelineNameInput struct {
```

Comment:
I think it's okay if we have these 3 fields be passed as parameters instead of creating a new reserved name in the `cli` pkg:
```go
func askDeployedPipelineName(selector codePipelineSelector, appName, msg string)
```
Maybe instead of taking a command as input we can also take the prompt msg for extra flexibility.

Reply:
Okay, I can make that change! 
But for the future: how do I know when it's better to make a struct and when it's better to use parameters? Is it that the struct is a more logical grouping of elements that belong together, but parameters are items that are needed but not really parts of a whole? And/or if the struct object will be used as such more than once?

Reply:
I should have written "nit" here I think - It's totally fine what you had before! I was just trying to reduce the number of names declared in the package since `cli` is so big 😅 

It's a good question, I declare a type when I can create a "data abstraction" and associate a set of behaviors (methods), or I want to create an "aggregate" type to bundle a bunch of related fields (like your situation or the AWS SDK).
Honestly, it was just a suggestion, it felt like the fn signature is small enough that we didn't need an extra type